### DISCUSSION TITLE
Standup (Monday, May 23rd)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#69] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```

**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Talked to @BrandonMagana  about middleware 
 - Took a look into how instantiating an array as an attribute of an entity

**What are your priorities for the next 24 hours?**:
 - Help @BrandonMagana solve the many to many relationship issue

**Blockers**:
 - Been a little busy with homework and some Atrevi stuff

**Shoutouts**:
 - @everyone, miss yall guys

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Autocomplete is now working (!!!)

**What are your priorities for the next 24 hours?**:
 - Clean code

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto I miss you

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Started learning about middlewares
 -  I'm currently implementing my own

**What are your priorities for the next 24 hours?**:
 - Finish auth middleware

**Blockers**:
 - Too many stupid elective course homework

**Shoutouts**:
 - @antonieto for helpping me with some backend issues

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - More MUI custom templates

**What are your priorities for the next 24 hours?**:
 - Implement them hopefully

**Blockers**:
 - Optatives evidence

**Shoutouts**:
 - @todalagentebonita

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Created the CRUD endpoints for aspirants
 - Chat with @antonieto about authentication

**What are your priorities for the next 24 hours?**:
 - Finish login and signup endpoints for aspirants

**Blockers**:
 - Lot of work

**Shoutouts**:
 - @antonieto for the authentication proposal
